flow/embedded_views.h

```diff
@@ -42,8 +56,88 @@ class ExternalViewEmbedder {
   virtual ~ExternalViewEmbedder() = default;
 
   FML_DISALLOW_COPY_AND_ASSIGN(ExternalViewEmbedder);
+
+#pragma mark - transforms
+  std::shared_ptr<FlutterEmbededViewTransformStack> transformStack;
+
+};  // ExternalViewEmbedder
+
+class FlutterEmbededViewTransformStack {
+ public:
+  void pushClipRect(const SkRect& rect);
+  void pushClipRRect(const SkRRect& rrect);
+  void pushClipPath(const SkPath& path);
+
+  void pushTransform(const SkMatrix& matrix);
+
+  // Removes the `FlutterEmbededViewTransformElement` on the top of the stack
+  // and destroys it.
+  void pop();
+
+  // Returns the iterator points to the bottom of the stack.
```

Comment:
Just from reading this comment I'm not sure what's the "bottom of the stack" the mutator that should apply to all other mutators in the stack, or the one that all other mutators applies to? can you make this more clear in the comment ?(for both iterators)

Reply:
Done. Updated to 
```  
// Returns the iterator points to the bottom of the stack.
// When we composite an embedded view, this is the first mutator we should apply to the view.
// And we should iterate through all the mutators until we reach `rend()` and apply all the mutations to the view along the way.
```
Let me know what you think.

Reply:
Suggestion:

```c++
// A stack of mutators that can be applied to an embedded platform view.
//
// The stack may include mutators like transforms and clips, each mutator applies to all the mutators that are below it in the stack
// and to the embedded view.
//
// For example consider the following stack: [T1, T2, T3], where T1 is the top of the stack and T3 is the bottom of the stack.
// Applying this mutators stack to a platform view P1 will result in T1(T2(T2(P1))).
class MutatorsStack {
...
// Returns an iterator pointing to the top of the stack.
std::vector<EmbeddedViewMutator>::reverse_iterator top();

// Returns an iterator pointing to the bottom of the stack.
std::vector<EmbeddedViewMutator>::reverse_iterator bottom();
}
```

Reply:
done